src/components/TopBar.jsx

```diff
@@ -48,6 +49,14 @@ const useStyles = makeStyles(theme => ({
 		width: '20px',
 		height: '20px',
 		padding: '0 4px 4px 0'
+	},
+	alert: {
+		width: '50%'
+	},
+	message: {
+		// marginLeft: '40px',
+		margin: 'auto',
+		textAlign: 'center'
```

Comment:
Since you outsourced the alert to its own component we don't need these additions anymore in `TopBar`.